src/frontend/utils/validators.ts

```diff
@@ -167,6 +167,7 @@ export const ikkeValider = <Value>(felt: FeltState<Value>): FeltState<Value> =>
 };
 
 export const erBegrunnelseGyldig = (felt: FeltState<string>, avhengigheter?: Avhengigheter) =>
-    avhengigheter?.erSkjønnsmessigVurdert && felt.verdi.length === 0
+    (avhengigheter?.erSkjønnsmessigVurdert || avhengigheter?.erMedlemskapVurdert) &&
+    felt.verdi.length === 0
         ? feil(felt, 'Du må skrive en begrunnelse ved skjønnsmessig vurdering.')
```

Comment:
Er vel denne som kanskje må forandres isåfall? Gir det ikke mening å kun huke av medlemskap og ikke skjønnsmessig vurdering?

Reply:
Tror det skal være mulig?